src/app/LocalizeModification.cpp

```diff
@@ -129,6 +129,7 @@ int LocalizeModificationApplication::main(int argc, char** argv) {
     VariableModTable* modTable = getModTable(match);
     MassConstants::Init(modTable->ParsedModTable(),
                         modTable->ParsedNtpepModTable(), modTable->ParsedCtpepModTable(),
+                        nullptr, nullptr,
```

Comment:
Comment by bmx8177: In src/app/LocalizeModification.cpp at line 130 you have two null pointers as input into MassConstants::init. Is this b/c you do not need to localize Nterm and Cterm mods?

Reply:
Correct. It also seems that LocalizeModifications is not used much, so figuring it out is a lower priority.